tests/sentry/workflow_engine/processors/test_workflow.py

```diff
@@ -400,3 +410,86 @@ def test_enqueue_workflow__adds_to_workflow_engine_set(
                 {"event_id": self.event.event_id, "occurrence_id": self.group_event.occurrence_id}
             ),
         )
+
+
+@django_db_all
+class TestDeleteWorkflow:
```

Comment:
why not subclass `BaseWorkflowTest` here so you can do `self.create_...` instead of having to do `Factories`?

Reply:
https://github.com/getsentry/sentry/pull/85509#discussion_r1962770446

so i can use parametrize -- `TestCase` breaks that.